build/makelib/golang.mk

```diff
@@ -156,6 +156,12 @@ go.test: go.vet go.fmt $(CGO_PREREQS)
 #	@$(GOHOST) test -v -i $(GO_PKG_FLAGS) $(GO_NONSTATIC_FLAGS) $(GO_ALL_PACKAGES)
 	@$(GOHOST) test -cover $(GO_PKG_FLAGS) $(GO_NONSTATIC_FLAGS) $(GO_ALL_PACKAGES)
 
+.PHONY: go.lint
+go.lint: $(GO_TOOLS_DIR)/golint
+	@lint=`$(GO_TOOLS_DIR)/golint $(GO_ALL_PACKAGES)`; \
```

Comment:
golint exits with exitcode 0 whether there are errors or not. So this was the way i could make it return a non-zero when it found errors. Does set_exit_status solve this?